### PR TITLE
Release 1.5

### PR DESCRIPTION
**Pull request by enadeau**
# Stravalib Release Pull Request Template

Please use this template when you are preparing to make a release to stravalib

* [An overview of our release workflow can be found in our documentation.](https://stravalib.readthedocs.io/en/latest/contributing/build-release-guide.html)
* [Before making a release be sure to check out test PyPI](https://test.pypi.org/project/stravalib/) to ensure that the build is working properly.

## Release checklist
- [x] Be sure to clearly specify what version you are bumping to in the PR title: Example: Bump to version x.x
- [x] Add the version of this release to our changelog
- [x] Organize items changes under the new version in groups as follows: Added, Fixed and Changed
- [x] Add all contributors to the release below those sections
- [x] Wait for a maintainer to approve the pull request. Then merge! You are now ready to create the release.

The changelog should look something like this:

```
## Unreleased

## New version here: e.g. v1.0.0

### Added
* Add: Add an option to mute Strava activity on update (@ollyajoke, #227)

### Fixed
* Fix: add new attributes for bikes according to updates to Strava API to fix warning message (@huaminghuangtw, #239)

### Changed
* Change: Improved unknown time zone handling (@jsamoocha, #242)

### Contributors to this release
@jsamoocha, @yihong0618, @tirkarthi, @huaminghuangtw, @ollyajoke, @lwasser
```

Once this PR is merged you are ready to

- [ ] Create a tagged release on GitHub using the same version that you merged in the changelog added here
- [ ] When you publish that release, the GitHub action to push to PyPI will be invoked.

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -2,14 +2,26 @@
 
 ## Unreleased
 
+## v.15
+
+### Added
+- Stravalib now includes types annotation, the package is PEP 561 compatible (@enadeau, #423)
 - Add: Add nox to run tests, build docs, build package wheel/sdist(@lwasser, #395, #397)
 - Type annotation for all file in the library (@enadeau, #384, #415)
```
Comment (lwasser) [inline changelog.md:10]:
```suggestion
- Type annotation for all files in the library (@enadeau, #384, #415)
```

File: changelog.md [hunk 2]
```diff
@@ -2,14 +2,26 @@
 
 ## Unreleased
 
-- Add: Add nox to run tests, build docs, build package wheel/sdist(@lwasser, #395, #397)
-- Type annotation for all file in the library (@enadeau, #384, #415)
-- Remove python 3.8 support following NEP-29 (@enadeau, #416)
+## v.15
```
Comment (lwasser) [inline changelog.md:5]:
```suggestion
## v1.5
```
we've been trying to be consist with our version using a lowercase v and then the version number.

**Review by lwasser — APPROVED** [review]
I made a few minor edits but this is good to go. you can merge and then create the release making sure the tag value is in this format:

v1.5

let me know if you run into any issues! but once you create and publish the tagged release a build and push action will begin to push the new v1.5 version to pypi 🚀

**Comment by lwasser** [comment]
ok @enadeau when you are back online, please merge this ! and then create a new tagged release. and 1.5 should be published shortly there after!! ✨

**Comment by lwasser** [comment]
here you can see the [previous tagged versions](https://github.com/stravalib/stravalib/tags) created.

**Comment by enadeau** [comment]
It's now live, thanks for the guidance!

**Comment by lwasser** [comment]
nicely done!! thanks @enadeau 🚀

**Pull request merged**